src/popup/components/ResultsTab.vue

```diff
@@ -1,21 +1,63 @@
 <template>
   <div class="tab results-tab">
+    <div class="tabs">
+      <button v-for="tab in tabs" :key="tab" class="tabs__action" v-bind:class="{'selected': activeTab === tab}" @click.prevent="changeTab(tab)">{{ tab }}</button>
```

Comment:
Would be cool if we could fit a little Puppeteer and Playwright icon just before the wording, i.e. "🎭 Playwright". Not sure if that translates well given the space we have in the tab.

Reply:
Will give a try

Reply:
@tnolet better?
![Screenshot 2020-08-05 at 19 58 40](https://user-images.githubusercontent.com/1178139/89447319-148a7900-d756-11ea-8e1c-aa70ce908d23.png)